sympy/functions/special/tests/test_error_functions.py

```diff
@@ -636,12 +636,12 @@ def test_si():
     assert tn_arg(Si)
     assert tn_arg(Shi)
 
-    assert Si(x)._eval_as_leading_term(x) == x
-    assert Si(2*x)._eval_as_leading_term(x) == 2*x
-    assert Si(sin(x))._eval_as_leading_term(x) == x
-    assert Si(x + 1)._eval_as_leading_term(x) == Si(1)
-    assert Si(1/x)._eval_as_leading_term(x, cdir=1) == \
-        Si(1/x)._eval_as_leading_term(x, cdir=-1) == Si(1/x)
+    assert Si(x)._eval_as_leading_term(x, None, 1) == x
+    assert Si(2*x)._eval_as_leading_term(x, None, 1) == 2*x
+    assert Si(sin(x))._eval_as_leading_term(x, None, 1) == x
+    assert Si(x + 1)._eval_as_leading_term(x, None, 1) == Si(1)
+    assert Si(1/x)._eval_as_leading_term(x, None, 1) == \
+        Si(1/x)._eval_as_leading_term(x, None, -1) == Si(1/x)
 
```

Comment:
If the convention is to pass `logx` and `cdir` as keyword arguments then we should do it here as well.

Also the cdir values should be sympified here like `S.One` or `S(1)`.